src/store/item-store.js

```diff
@@ -1,26 +1,23 @@
 import { Store } from 'consus-core/flux';
-
+import { searchItem } from '../lib/api-client.js';
 let item = null;
 
 class ItemStore extends Store {
     getItem() {
-        if (item === null) {
-            return item;
-        }
-        return {
-            id: item.id,
-            status: item.status
-        };
+        return item;
+    }
+    getAllItems(){
+        return item;
+    }
+    searchItemByAddress(address){
```

Comment:
Fair enough, I questioned where to put this call I can move it to the view. It just seemed wrong to do an api call from the view for me.